nextflow_schema.json

```diff
@@ -483,6 +493,117 @@
+                    "type": "integer",
+                    "default": 10000000,
+                    "description": "Global maximum query length of DNA strand.",
+                    "minimum": 10
+                },
+                "lx_max_aa_len": {
+                    "type": "integer",
+                    "default": 10000000,
+                    "description": "Gloval maximum query length of AminoAcid strand.",
+                    "minimum": 10
+                },
+                "lx_min_dna_ident": {
+                    "type": "number",
+                    "default": 80,
+                    "description": "Global minimum DNA percent identity required for match. (Float value)",
+                    "minimum": 0,
+                    "maximum": 100
+                },
+                "lx_min_aa_ident": {
+                    "type": "number",
+                    "default": 80,
+                    "description": "Global minimum minimum AminoAcid percent identiy required for match. (Float value)",
+                    "minimum": 0,
+                    "maximum": 100
+                },
+                "lx_min_dna_match_cov": {
+                    "type": "number",
+                    "default": 80,
+                    "description": "Gloval minimum DNA percent hit coverage identity required for match. (float)",
+                    "minimum": 0.001,
+                    "maximum": 100
+                },
+                "lx_min_aa_match_cov": {
+                    "type": "number",
+                    "default": 80,
+                    "description": "Global minimum AminoAcid hit coverage identity required for match. (float)",
+                    "minimum": 0,
+                    "maximum": 100
+                },
+                "lx_max_target_seqs": {
+                    "type": "number",
+                    "default": 10,
+                    "description": "Maximum number of sequence hits per a query",
+                    "minimum": 1
+                },
+                "lx_extraction_mode": {
+                    "type": "string",
+                    "default": "raw",
+                    "description": "Different ways to run locidex.",
+                    "enum": [
+                        "snps",
+                        "trim",
+                        "raw",
+                        "extend"
+                    ]
+                },
+                "lx_report_mode": {
+                    "type": "string",
+                    "default": "normal",
+                    "description": "Allele profile assignment.",
+                    "enum": [
+                        "normal",
+                        "conservative"
+                    ]
+                },
+                "lx_report_prop": {
+                    "type": "string",
+                    "default": "locus_name",
+                    "description": "Metadata label to use for aggregation. Only underscores are allowed in sample names",
```

Comment:
Could you list all the allowed characters "alphanumeric, underscores, and dashes"?

Reply:
updated in:  8aaba2b